### DISCUSSION TITLE
Suggestion for a use case

### DISCUSSION THREAD

**Discussion by arne1921KF** [teunbrand/ggchromatic#15] [Ideas]
I've seen a practical use case [published in 2016](https://doi.org/10.1111/ele.12624). (OA pre-print marked CC-BY including the figures [here](https://dx.doi.org/10.7287/peerj.preprints.1326v2) .)

Fig. 1 explains the framework, Fig. 2 & Fig. 3 make a nice use of a colour space visualising distances other than geographic distances on a map. Fig. 4 is basically a model case for something which your package could be useful for.

Plotting multivariate data reduced to three dimensions (via ordination methods) on _another_ figure (as a map, as shown in Meyer et al. 2016) actually can make a lot of sense in visualisation. The approach has, of course, some inherent problems. However, I find it quite intuitive and thus compelling.

**Comment by teunbrand** [comment]
Yeah these seem good use cases to map values to different dimensions of a colourspace. Colour is never going to be a perfect tool to convey quantitative information, considering how many philosophies are around even about simple sequential colour scales, but it conveys intuition quite appropriately I think.
